src/main/java/com/epam/aidial/cfg/client/dto/ToolSetResourceDto.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.aidial.cfg.client.dto;
+
+import com.fasterxml.jackson.databind.PropertyNamingStrategies;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+import java.util.List;
+import java.util.Map;
+
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+@JsonNaming(PropertyNamingStrategies.SnakeCaseStrategy.class)
+public class ToolSetResourceDto {
+
+    private String name;
+    private String endpoint;
+    private String displayName;
+    private String displayVersion;
+    private String iconUrl;
+    private String description;
+    private String reference;
+    private Boolean forwardAuthToken;
+    private FeaturesDto features;
+    private List<String> inputAttachmentTypes;
+    private Integer maxInputAttachments;
+    private Map<String, Object> defaults;
+    private List<String> interceptors;
```

Comment:
let's remove/commentOut interceptors, dependencies, features, forwardAuthToken, reference, inputAttachmentTypes, maxInputAttachments and defaults properties. actually it's not used by core for toolset for now

Reply:
done